deliverables/README.md

```diff
@@ -1,8 +1,11 @@
 ##Week 2: Introduction to Ruby
 ###Intro to the Back End
 ####Required
-What is the difference between front- and back-end development?<br>
-Why do you need to install Ruby but not HTML/CSS?
+What is the difference between front- and back-end development?  
+As the name states “Front-End” is the part of the code that is on the front of the application. It is usually visible to user in the form of an interface inviting to interact with user. The main purpose of the front-end code is to interact with user, as well as present the data in a well-defined style and matter. The back-end development is a part of the application that is never visible to the user. It is built with use of server-side language and database. Front-end code interacts with a user in real time while back-end code interacts with a server to return user ready results. Anything displayed on the website is because of the query performed on the server returned data to the front-end.<br>
+
+Why do you need to install Ruby but not HTML/CSS?  
+We need RUby becasue it is the programing language that tells our website what to do. It is the interpretive language that truly controls the website. where HTML/CSS is more of the skelton of the wesite, which just shows whats on each page so it is the design adn lay out of the site.     
```

Comment:
It's not actually due to some difference in the nature or role of the languages.  The answer is simpler:  by having a browser like Chrome, you *already have* an HTML/CSS/Javascript interpreter installed.  That's why you don't need to install one.